diff -Naur vega-cli/src/args.js vega-cli-patched/src/args.js
--- vega-cli/src/args.js	1985-10-26 01:15:00.000000000 -0700
+++ vega-cli/src/args.js	2019-07-18 12:19:39.000000000 -0700
@@ -39,6 +39,10 @@
       .describe('h', 'Include XML header and SVG doctype.');
   }
 
+  args.boolean('vl')
+    .alias('vl', 'vegaLite')
+    .describe('vl', 'Compile spec as Vega-Lite.');
+
   args.number('s')
     .alias('s', 'scale')
     .default('s', 1)
diff -Naur vega-cli/src/render.js vega-cli-patched/src/render.js
--- vega-cli/src/render.js	1985-10-26 01:15:00.000000000 -0700
+++ vega-cli-patched/src/render.js	2019-07-18 12:19:39.000000000 -0700
@@ -1,4 +1,5 @@
 const vega = require('vega'),
+      vl = require('vega-lite'),
       path = require('path'),
       args = require('./args'),
       read = require('./read');
@@ -44,6 +45,11 @@
 
   // instantiate view and invoke headless render method
   function render(spec) {
+    // use vega-lite, if specified
+    if (arg.vegaLite) {
+      spec = vl.compile(spec, { logger: vega.logger(loglevel, 'warn') }).spec;
+    }
+
     const view = new vega.View(vega.parse(spec, config), {
         loader: vega.loader({baseURL: base}),   // load files from base path
         logger: vega.logger(loglevel, 'error'), // route all logging to stderr
